packages/web-twig/tests/ComponentSnapshotTest.php

```diff
@@ -61,6 +61,7 @@ private function formatHtml(string $html): string
         $config = [
             'indent' => true,
             'output-xhtml' => true,
+            'vertical-space' => true,
```

Comment:
Try adding also `indent-attributes`...

Reply:
https://api.html-tidy.org/tidy/quickref_5.8.0.html#indent-attributes

Reply:
I tried, but didn't see any result… I'll try one more time 🙂.

Reply:
Oh, even worse: it joins some lines into one…

![image](https://github.com/lmc-eu/spirit-design-system/assets/5614085/41ecba96-a7fd-4849-b872-a63e8076a62f)

I wouldn't do that 🙂.